scss/typography/_base.scss

```diff
@@ -441,3 +433,26 @@ $abbr-underline: 1px dotted $black !default;
     }
   }
 }
+
+/// Calculates the header font-size based on breakpoints.
+///
+/// @param {String} the size of the header to output.
+@mixin typography-header-font-size (
+  $requested-size
+) {
```

Comment:
Not sure I'm fully understanding the intended behavior from requested-size... looks like we still iterate through all sizes and breakpoints and output them, so what exactly is the intent?